docs/2020-06-isc20/contributing.md

```diff
@@ -83,13 +83,13 @@ a project that aims to *automate* tedious software installation procedures.
 
 At the end of 2015 efforts were made to tackle this issue by implementing
 GitHub integration features in EasyBuild, which automate the contribution
-workflow by running `git` commands and interacting with the [GitHub API](https://developer.github.com/v3/).
+workflow by running `git` commands and interacting with the [GitHub API](https://blogs.oracle.com/linux/post/attack-of-the-cosmic-rays).
```

Comment:
@klust This replacement in faulty? (it was done correctly in `docs/2021-isc21/contributing.md` though)

Reply:
Indeed faulty and corrected in a new commit. That's what happens when you switch all the time between normal computers and a Mac with a different key combination for copy and paste and update a lot of links one after another...